app/guid-node/forks/controller.ts

```diff
@@ -0,0 +1,102 @@
+import { action } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Controller from '@ember/controller';
+import { task } from 'ember-concurrency';
+
+import I18N from 'ember-i18n/services/i18n';
+import Node from 'ember-osf-web/models/node';
+import StatusMessages from 'ember-osf-web/services/status-messages';
+import Toast from 'ember-toastr/services/toast';
+
+export default class GuidNodeForks extends Controller {
+    @service toast!: Toast;
+    @service i18n!: I18N;
+    @service statusMessages!: StatusMessages;
+
+    toDelete: Node | null = null;
+    deleteModal = false;
+    newModal = false;
+    page = 1;
+    forks = [];
+    maxPage: number = 1;
+    perPage = 10;
+
+    getForks = task(function* (this: GuidNodeForks) {
+        const page = this.get('page');
+        const model = this.get('model');
+        const node = yield model.taskInstance;
+        const forks = yield node.queryHasMany('forks', { page });
```

Comment:
Might be worth embedding contributors to avoid piecemeal loading. Either that or a skeleton screen.

...actually, this whole page should maybe use a skeleton screen, instead of the `ball-scale`. @brianjgeiger, what's our stance on that?

Reply:
In general, pro-skeleton screen. In cases like this, I don't know. If we know going in or early how many forks we have, then we can make an effective skeleton. If not, how many fake boxes do we show?

Reply:
Makes sense. I'm good with the loading indicator, but do want contributors embedded. I hate jumpy UIs.

Reply:
Small problem (maybe not a problem). @brianjgeiger, can we filter embeds? For some unknown reason (would be great if we changed it tbh) forks only list "bibliographic" contributors, which is  a subset of all contributors.